extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss

```diff
@@ -0,0 +1,137 @@
+@import 'skins/shared/color';
+@import 'skins/oasis/css/core/breakpoints-variables';
+
+$color-blue-darken: #144b93;
+$color-blue: #1a5eb8;
+$color-message-text: #333;
+$close-button-background: '/resources/wikia/ui_components/modal/images/close-dark.svg';
+
+@if( $is-dark-wiki ) {
+	$color-message-text: #fff;
+	$close-button-background: '/resources/wikia/ui_components/modal/images/close-light.svg';
+}
+
+.pmp-entry-point {
+	background: url('/extensions/wikia/PotentialMemberPageExperiments/images/858-px-background.png') no-repeat center bottom;
+	border-bottom: 2px solid $color-blue;
+	height: 150px;
+	margin: 18px 0;
+	overflow: hidden;
+	position: relative;
+	text-align: center;
+
+	p.pmp-entry-point-message {
```

Comment:
Prefixing a class with its element selector [should be avoided](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#specificity-and-nesting). Is this needed for any reason?